style.css

```diff
@@ -7,3 +7,125 @@
 html {
   font-size: 62.5%;
 }
+
+body {
+  font-family: "Inter", sans-serif;
+  width: 37.5rem;
+  margin: 0 auto;
+  line-height: 1;
+  background-color: #e5e5e5;
+}
```

Comment:
- [ ] Please don't use fix `width` because it will make problems in `responsiveness` so kindly don't use fix `width` :slightly_smiling_face: